packages/tx/src/capabilities/eip2718.ts

```diff
@@ -0,0 +1,26 @@
+import { RLP } from '@ethereumjs/rlp'
+import { concatBytes } from '@ethereumjs/util'
+import { keccak256 } from 'ethereum-cryptography/keccak.js'
+
+import { txTypeBytes } from '../util.js'
+
+import { errorMsg } from './legacy.js'
+
+import type { EIP2718CompatibleTxInterface, TypedTransaction } from '../types'
+import type { Input } from '@ethereumjs/rlp'
+
+export function getHashedMessageToSign(tx: EIP2718CompatibleTxInterface): Uint8Array {
+  return keccak256(tx.getMessageToSign())
+}
+
+export function serialize(tx: EIP2718CompatibleTxInterface, base?: Input): Uint8Array {
+  return concatBytes(txTypeBytes(tx.type), RLP.encode(base ?? tx.raw()))
+}
+
+export function validateYParity(tx: TypedTransaction) {
```

Comment:
Is it not possible to type with `EIP2718CompatibleTxInterface` here?

Reply:
Forgot to adjust after refactoring, fixed.